### PR TITLE
v88.0.0

### PR DESCRIPTION
**Pull request by romoore**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,211 @@
...
+*removed* property: `mdc_legacyFontScaling` in `MDCButton`
+
+### Snackbar
+
+#### MDCSnackbarManager
+
+*new* property: `adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable` in `MDCSnackbarManager`
+
+#### MDCSnackbarMessageView()
+
```
Comment (codeman7) [inline CHANGELOG.md:79]: I think we can remove the () here and below

**Review by codeman7 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (romoore) [reply CHANGELOG.md:79]: That's how it's generated in the API Diff tool. I'd prefer to either leave it as-is (to remain consistent from release to release) or remove it entirely (which I did for some silly garbled nonsense).

Comment (codeman7) [reply CHANGELOG.md:79]: Sounds good.

Comment (romoore) [reply CHANGELOG.md:79]: Which would you like me to do? Remove the API Diff or leave it as-is?

Comment (romoore) [reply CHANGELOG.md:79]: Spoke offline and agreed to leave it as-is for this release.

**Pull request merged**